### PR TITLE
Added Indonesian Translate

### PR DESCRIPTION
**Pull request by barraIhsan**
Added Indonesian (`id_id.json`) Translate

### CODE REVIEW

File: src/main/resources/assets/seedmapper/lang/id_id.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "commands.exceptions.blockNotFound": "Block \"%s\" tidak ditemukan.",
+
+  "arguments.exceptions.tooFewArguments": "Argumen terlalu sedikit.",
+
+  "command.config.setAutomate": "Set \"otomatis\" menjadi %s.",
+  "command.config.getAutomate": "\"otomatis\" saat ini diatur ke %s.",
+  "command.config.toggleAutomate": "Aktifkan \"otomatis\" menjadi %s.",
+  "command.config.setSeed": "Set \"benih\" menjadi %s.",
+  "command.config.getSeed": "\"benih\" saat ini diatur ke %s.",
+  "command.config.getSeed.null": "\"benih\" adalah null",
```
Comment (xpple) [inline src/main/resources/assets/seedmapper/lang/id_id.json:19]: There should be a `.` here.

**Review by xpple — CHANGES_REQUESTED** [review]
Thank you! Just one little thing :).

File: src/main/resources/assets/seedmapper/lang/id_id.json (hunk 1 above)
Comment (barraIhsan) [reply src/main/resources/assets/seedmapper/lang/id_id.json:19]: In line 19?

Comment (xpple) [reply src/main/resources/assets/seedmapper/lang/id_id.json:19]: Yep.

Comment (barraIhsan) [reply src/main/resources/assets/seedmapper/lang/id_id.json:19]: ok

Comment (barraIhsan) [reply src/main/resources/assets/seedmapper/lang/id_id.json]: Done

**Review by xpple — APPROVED** [review]

**Comment by xpple** [comment]
Thank you!

**Pull request merged**